pkg/tfvars/openstack/openstack.go

```diff
@@ -166,15 +166,6 @@ func TFVars(masterConfigs []*v1alpha1.OpenstackProviderSpec, cloud string, exter
 		if err != nil {
 			return nil, err
 		}
-
-		// Make sure that the network has the primary cluster network tag.
-		// In the case of multiple networks this tag is required for
-		// cluster-api-provider-openstack to define which ip address to set as
-		// the primary one.
-		err = setNetworkTag(cloud, cfg.MachinesNetwork, infraID+"-primaryClusterNetwork")
```

Comment:
Shouldn't this function definition also get removed?

Perhaps the cluster destroy[1] needs clean up as well.
[1] https://github.com/openshift/installer/blob/9be316bc89f560653e3ad07cde7a716bf938cf6d/pkg/destroy/openstack/openstack.go#L1224-L1225

Reply:
Absolutely, thanks for noticing.

Reply:
generally speaking there are a lot of things to remove...